frontend-opal/src/thunks/showtrade.js

```diff
@@ -0,0 +1,11 @@
+import actions from 'actions/showTrade';
+
+function showTradeName(input) {

```

Comment:
@martinupset this is not needed to be a thunk just create a simple action

Reply:
ok